.github/.kodiak.toml

```diff
@@ -0,0 +1,4 @@
+version = 1
+merge.blocking_labels = ["WIP", "blocked"]
+merge.notify_on_conflict = false
```

Comment:
this is mostly so kodiak doesn't remove the `automerge` on conflicts. mostly useful for dependency updates where the update bot will resolve the conflict.